test/utils.ts

```diff
@@ -47,11 +47,13 @@ export function assertValidSchema(
         const actual: any = JSON.parse(JSON.stringify(schema));
 
         // uncomment to write test files
-        // writeFileSync(
-        //     resolve(`${basePath}/${relativePath}/schema.json`),
-        //     JSON.stringify(schema, null, 4) + "\n",
-        //     "utf8"
-        // );
+        if (process.env.UPDATE_SCHEMA) {
+            writeFileSync(
+                resolve(`${basePath}/${relativePath}/schema.json`),
+                JSON.stringify(schema, null, 4) + "\n",
```

Comment:
Let's use `import stringify from "json-stable-stringify";`. It's what we use by default to generate schema files already.